seed/static/seed/js/filters/district.js

```diff
@@ -12,7 +12,7 @@ angular.module('district', []).filter('district', function () {
     if (_.isNil(input)) {
       return input;
     }
-    if (angular.uppercase(input) === 'DISTRICT') {
+    if (input.toUpperCase() === 'DISTRICT') {
```

Comment:
interesting change...

Reply:
Yeah, that Angular function was deprecated since `toUpperCase` is now a first class citizen in JavaScript